### ISSUE TITLE
[Bug] This white circle is ugly!

### ISSUE THREAD

**Issue by mackeev** [fork-maintainers/iceraven-browser#219]
I definitely didn't expect the icon to look like this. Can we get rid of the circle?
Otherwise we should rename it to Icefoetus.
![Screenshot_Lawnchair_20201015-020747](https://user-images.githubusercontent.com/40295425/96055634-42461980-0e8d-11eb-84fb-0df259389d36.png)

**Comment by abhijitvalluri** [comment]
Well, if you did not like it, you should have had your voice heard in the poll if you haven't. Otherwise, unfortunately for you and anyone who does not like it this is what people chose. We can and will update the icon when we get better designs. I had to pick a light colored circle like white to make the logo's shape easy to identify due to its fairly dark colors.

**Comment by mackeev** [comment]
I cast my vote for the rightmost icon on top. You can read my comment for #199. I didn't vote for foetus. It looks fine on the About screen.
This is called bait-and-switch.
![Screenshot_20201015-150223386__01](https://user-images.githubusercontent.com/40295425/96129252-119ec800-0eff-11eb-8578-90ef7f738ca2.jpg)

**Comment by abhijitvalluri** [comment]
Ummm, the icon used for the app is exactly the same as the right most one above. And we need to put this icon within a background layer that defines the icon's shape. This is the "white circle" you are talking about. Please look carefully. It is the same icon.

**Comment by bajjou13214** [comment]
> Ummm, the icon used for the app is exactly the same as the right most one above. And we need to put this icon within a background layer that defines the icon's shape. This is the "white circle" you are talking about. Please look carefully. It is the same icon.

I think he wants the white circle removed because it makes the icon look like a foetus (on closer look at the icon i agree lmao)

**Comment by mackeev** [comment]
Exactly, my beef's with the background layer. It should be transparent.

**Comment by klaxosaur** [comment]
I agree, this is **not** the icon that the community voted for **unless** the white border is removed, it looks unsightly.

**Comment by reeseovine** [comment]
old issue, i know, but i have something to add on this topic. to me the white circle makes it looks ugly because the raven's outside edge is a bit wobbly, so the negative space ends up being uneven and unsightly. i don't see the fetus though 😅

i propose slightly modifying it by nudging the path nodes to fit better inside a circle. a whole new logo like proposed in #224 seems like the wrong way to go, since most users would probably be caught off guard by the sudden change.

i can do that and open a PR if there's interest.

**Comment by reeseovine** [comment]
ok that wasn't too hard... how's this?

<img width="1080" height="1080" alt="Image" src="https://github.com/user-attachments/assets/2818b7bb-8863-464f-a98e-27bcd949c6b7" />

**Comment by ArghItsDaveyy** [comment]
It's not hard to do at all, but I still see the white background/your image is the same one the OP was complaining about as far as I can see?

Here is what I believe OP wanted:

<img width="1491" height="1406" alt="Image" src="https://github.com/user-attachments/assets/c014ef34-d8f2-44ba-a744-9767efe93bb6" />

Either way I am impartial because I would honestly prefer a new icon altogether, the current icon doesn't look like a bird to me.
